de.renew.Renew.yml

```diff
@@ -0,0 +1,40 @@
+id: de.renew.Renew
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: loadrenew
+finish-args:
+  - "--env=PATH=/app/jre/bin:/usr/bin:/app/bin"
+  - "--share=network"
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--filesystem=host" 
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: renew
+    buildsystem: simple
+    build-commands:
+      - unzip -d /app/renew/ renew4.1base.zip
+      - export PATH="/app/jre/bin:/usr/bin:/app/bin"; cd /app/renew/bin/unix; yes n | ./installrenew
+      - install -Dm755 /app/renew/bin/unix/loadrenew /app/bin/loadrenew
+      - mkdir -p /app/share/icons/hicolor/
+      - cp -r /app/renew/bin/unix/freedesktop/icons/* /app/share/icons/hicolor/
+      - for i in $(find /app/share/icons/hicolor/ -type f -name 'application*.png'); do mv "$i" "$(dirname $i)/de.renew.Renew.$(basename $i)"; done
+      - for i in $(find /app/share/icons/hicolor/ -type f -name 'renew.png'); do mv "$i" "$(dirname $i)/de.renew.Renew.png"; done
+      - sed -i 's#\(<mime-type type="application/\(.*\)">\)#\1\n         <icon name="de.renew.Renew.application-\2" />#' /app/renew/bin/unix/freedesktop/renew.xml
+      - sed -i 's#^Icon=.*$#Icon=de.renew.Renew#' /app/renew/bin/unix/freedesktop/Renew.desktop
```

Comment:
please use `desktop-file-edit` to modify desktop files.

Reply:
Okay, thanks for the remark. I will look into it.

Reply:
I now modify the desktop file with desktop-file-install